packages/hub/src/scope.ts

```diff
@@ -125,7 +125,7 @@ export class Scope {
    * @param level string {@link Severity}
    */
   public setLevel(level: Severity): Scope {
-    this.level = level;
+    this.level = standardizeAny(level);
```

Comment:
Same here, do we really need this here?

Reply:
No and no. Just wanted to unify it JUST in case someone passes circular object as the fingerprint or level, as they are public API. I know it'll most likely never happen, but there's almost 0 cost to this and we won't blow up in the face to anyone :p